src/gui/dialogs/ShortcutDialog.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import {lang} from "../../misc/LanguageViewModel"
+import m from "mithril"
+import stream from "mithril/stream/stream.js"
+import {Dialog} from "../base/Dialog"
+import {Keys} from "../../api/common/TutanotaConstants"
+import {TextFieldN} from "../base/TextFieldN"
+import type {Shortcut} from "../../misc/KeyManager"
+import {ButtonType} from "../base/ButtonN"
+
+let isOpen = false
```

Comment:
I would prefer if this was being tracked in the `KeyManager` rather than globally here, but also maybe YAGNI?